README.md

```diff
@@ -1,65 +1,86 @@
-# Welcome to Streamlit :wave:
+![Streamlit logo|200x200](https://i.ibb.co/5FdSr1V/streamlit.png)
 
-**The fastest way to build and share data apps.**
+# Streamlit
 
-Streamlit lets you turn data scripts into shareable web apps in minutes, not weeks. It’s all Python, open-source, and free! And once you’ve created an app you can use our [Community Cloud platform](https://streamlit.io/cloud) to deploy, manage, and share your app!
+**A faster way to build and share data apps.**
+
+Streamlit lets you turn data scripts into shareable web apps in minutes, not weeks. It’s all Python, open-source, and free! And once you’ve created an app you can use our [Community Cloud platform](https://streamlit.io/cloud)  to deploy, manage, and share your app!
 
-![Example of live coding an app in Streamlit|635x380](https://raw.githubusercontent.com/streamlit/docs/main/public/images/Streamlit_overview.gif)
 
 ## Installation
 
+Open a terminal and run:
 ```bash
-pip install streamlit
-streamlit hello
+$ pip install streamlit 
+$ streamlit hello
 ```
 
-Streamlit can also be installed in a virtual environment on [Windows](https://github.com/streamlit/streamlit/wiki/Installing-in-a-virtual-environment#on-windows), [Mac](https://github.com/streamlit/streamlit/wiki/Installing-in-a-virtual-environment#on-mac--linux), and [Linux](https://github.com/streamlit/streamlit/wiki/Installing-in-a-virtual-environment#on-mac--linux).
+This should open our sweet _Streamlit Hello_ app right in your browser.
```

Comment:
maybe something like "...our sweet _Streamlit Hello_ app with a bunch of examples...". May be a bit unclear what that app is. (Then again, might not matter much...)

Reply:
"This should open our sweet _Streamlit Hello_ app right in your browser. It features a bunch of examples of what you can do with Streamlit!"